torch/hub.py

```diff
@@ -2,18 +2,14 @@
 import os
 import shutil
 import sys
-import tempfile
 import zipfile
 
 if sys.version_info[0] == 2:
     from urlparse import urlparse
     from urllib2 import urlopen  # noqa f811
 else:
     from urllib.request import urlopen
-    from urllib.parse import urlparse
-
-import torch
-import torch.utils.model_zoo as model_zoo
```

Comment:
@ailzhang Was this import intentional (to reexpose it as `torch.hub.model_zoo`?) If so, I can add it back.

Reply:
Yea that was one confusion we had about model_zoo and hub I think. I have a meeting with @soumith in 2hrs. I will bring this up for discussion during the meeting and get back to you here. Thanks!

Reply:
@ezyang Feel free to remove them for now. I will make a followup PR to redirect `torch.utils.model_zoo` properly to `torch.hub` with corresponding doc changes. No need to block this PR from merging. Thanks!